atomate/vasp/firetasks/run_calc.py

```diff
@@ -147,6 +148,11 @@ def run_task(self, fw_spec):
         elif job_type == "full_opt_run":
             jobs = VaspJob.full_opt_run(vasp_cmd, auto_npar=auto_npar, ediffg=self.get("ediffg"),
                                         max_steps=5, half_kpts_first_relax=False)
+        elif job_type == "neb":
```

Comment:
Note: prior to the NEB commits, half_kpoint_first was hardcoded to False for the other custodian types. I am pretty sure that at some time, this parameter was defaulted to True in custodian (which I did not like since I am a big fan of principle of least surprise and "do no harm" for default parameters) and so in atomate I defaulted it to False so there were no surprises of a user's kpoint mesh changing. Since then, custodian now has half_kpoint_first = False as the default and there is no reason to enforce this in atomate. It is for the same reason that atomate sets auto_npar to False since for >1 node jobs this parameter can lead to unintended and confusing behavior when the user is trying to set NPAR themselves (as we saw from some user reports) yet never explicitly told custodian to change their NPAR.

For the moment, @HanmeiTang should make two changes. (i) document all new parameters as optional parameters and (ii) make sure the new optional parameters get used for all job types not just NEB (with the same defaults as before).

In the future, I will probably try to clean up some of this with a more **kwargs style approach so there isn't a need to add a new parameter to atomate every time there is a new custodian option. The only thing atomate may want to do is to deflect some of the default behavior of custodian to be more in line with atomate's principles for how workflows should operate by default.